site/reference/schema/kptfile/README.md

```diff
@@ -0,0 +1,140 @@
+# Example kptfile
```

Comment:
So either:
1) Provide the schema (not an example)
2) Provide a valid example using realistic values

If we're doing 1, then doing something in the spirit of existing site:
https://googlecontainertools.github.io/kpt/api-reference/kptfile/
but improved:
- Styling (font, color, etc) needs to be fixed
- We should provide example values in the comments for each field (See later comment on realistic example values)

If we're doing 2, then there are bunch of issues:
1. Whitespacing (remove blank lines)
2. Use an example (e.g. `nginx` or `wordpress`) following the book to show real values.

@droot has opinions/preferences here as well.